x-pack/test/functional/services/ml/common_table_service.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { WebElementWrapper } from '../../../../../test/functional/services/lib/web_element_wrapper';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export type MlTableService = ReturnType<typeof MlTableServiceProvider>;
+
+export function MlTableServiceProvider({ getPageObject, getService }: FtrProviderContext) {
```

Comment:
Decided to add a reusable service as we have many EUI tables and at the moment there is a lot of copy-pasted code.